Dockerfile

```diff
@@ -0,0 +1,16 @@
+# declaration of NEXUS_VERSION must appear before first FROM command
+# see: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
+ARG NEXUS_VERSION=latest
+
+FROM maven:3-jdk-8-alpine AS build
+
+COPY . /nexus-repository-r/
+RUN cd /nexus-repository-r/; \
+    mvn clean package -PbuildKar;
+
+FROM sonatype/nexus3:$NEXUS_VERSION
```

Comment:
I could be wrong but looks like {} is missed

Reply:
From our session last week, this is working as it, but were you able to get it working on your machine after pruning, etc?

Reply:
Hi @bhamail . Sorry for so long response. Will try to finish my verification today.

Reply:
I couldn't make this fail as is. Did you have any luck verifying it?

Reply:
it works and ARG variable is ok.